keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -292,29 +546,245 @@ N/A
 
 N/A
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+It's up to the user what package repository (OBS or Google) they want to use.
```

Comment:
> It's up to the user what package repository (OBS or Google) they want to use.

We need to define more clearly when we would - or wouldn't - direct folks to this new repo.

Reply:
This is out-of-scope of this KEP for now because it's way too early to decide on that. It'll be part of the communications plan.

Reply:
Once we graduate this KEP and the OBS implementation to beta, we'll strongly recommend using the OBS repos to take the load from Google. The Google repos will be deprecated in the future, so we need to consider that too. I'll clarify this in the KEP.